frontend/packages/ceph-storage-plugin/integration-tests/views/pvc.view.ts

```diff
@@ -18,6 +18,11 @@ export const selectAccessMode = (accessMode) => $(`input[value=${accessMode}]`);
 export const inputPVCSize = $('[name=requestSizeValue]');
 export const sizeUnitsDropdown = $('[data-test-id=dropdown-button]');
 
+// expand pvc
+export const expandButton = element(by.buttonText('Expand'));
+export const capacityDropdown = $('[data-test-id="dropdown-button"]');
```

Comment:
It should be named as `capacityUnitDropdown` as its the dropdown for units if I am not mistaken.

Reply:
Yes the variable name is incorrect.